### PR TITLE
updated rook version support

### PR DESCRIPTION
**Pull request by Rookeret73**
Ticket #1965
Rook setup, Rook reference, Rook troubleshooting

Java 7,8 (Oracle, OpenJDK)
Python 2.7, 3.5, 3.6, 3.7 (C-Python, Pypy)
Node 4.3, 6, 8, 10

### CODE REVIEW

File: docs/installation-node.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Node
 ## Adding a Node.js Rook
 
 __Pre-requisites:__  
-- *Node v4.x/v6.x/v8.x* ([download here](https://nodejs.org/))
+- *Node v4.3/v6.x/v8.x/v10.x* ([download here](https://nodejs.org/))
```
Comment (LiranLast) [inline docs/installation-node.md]: 4.3 +

File: docs/rooks-node.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ const rook = require('rookout/auto_start');
 
 ## Supported Versions
 
-| Implementation     | Versions |
-| ------------------ | -------- |
-| **Node**           | 4, 6, 8  |
+| Implementation     | Versions       |
+| ------------------ | -------------- |
+| **Node**           | 4.3, 6, 8, 10  |
```
Comment (LiranLast) [inline docs/rooks-node.md]: 4.3 +

Can we somehow share this between files?

Comment (Rookeret73) [reply docs/rooks-node.md]: Added 4.3+ and pushed.
I will check if the docs framework can have a shared "variable".

**Comment by Rookeret73** [comment]
@orishavit 
Hey, as we discussed, can you please approve again?
Thanks :)

**Pull request merged**